.github/workflows/tag-latest-ontology-assets.yml

```diff
@@ -0,0 +1,24 @@
+name: Tag 'Latest' Ontology Assets
+
+on:
+  push:
+    paths:
+      - "**/ontology-assets/**.json"
```

Comment:
We should change this to trigger on `"- **/ontology-assets/**decendants.json"` and remove trigger on `- "**/ontology-assets/**.gz"`

Reply:
we might want it to trigger if other assets get updated as well, though--in case a frontend ever needs to fetch a latest curated list, for instance. Or do you think we should just wait until we get to that